OrbitLinuxTracing/UprobesReturnAddressManager.h

```diff
@@ -59,6 +61,75 @@ class UprobesReturnAddressManager {
     }
   }
 
+  // In case of callchain sampling we don't have the complete stack to patch,
+  // but only the callchain (as list of instruction pointers). In those,
+  // a uprobe address occurs instead of the caller of an instrumented function.
+  // This function patches the callchain, using the maps information to identify
+  // instruction pointers of uprobe code and using the return address safed in
+  // the uprobes.
+  bool PatchCallchain(pid_t tid, uint64_t* callchain, uint64_t callchain_size,
+                      unwindstack::Maps* maps) {
+    if (!tid_uprobes_stacks_.contains(tid)) {
+      return true;
+    }
+
+    auto& tid_uprobes_stack = tid_uprobes_stacks_.at(tid);
+    CHECK(!tid_uprobes_stack.empty());
+
+    std::vector<uint64_t> ips_to_patch;
```

Comment:
The content is not IPs, I would call it something more generic like `frames_to_patch`.

Reply:
Sure